Parser/Parser.php

```diff
@@ -65,18 +83,26 @@ public function parseDirectory(string $rootDirectoy, OutputInterface $output = n
             $progressBar->finish();
         }
 
+        $this->logErrors($output);
+
         return $calls;
     }
 
     /**
      * parse will parse the given $contents as php code and emit all calls to the symfony container `get` method.
      *
      * @param string $contents
+     * @param string $fileName
      * @return CallToContainer[]
      */
     public function parse(string $contents, string $fileName): array
     {
-        $ast = $this->parser->parse($contents);
+        try {
+            $ast = $this->parser->parse($contents);
+        } catch (Error $e) {
+            $this->errors[] = sprintf("Could not parse file %s: %s", $fileName, $e->getMessage());
```

Comment:
Single quotes for strings (also, in a couple of other places)

Reply:
done

Reply:
:ok: